app/assets/stylesheets/landing.scss

```diff
@@ -63,6 +64,45 @@
   }
 }
 
+.footer-section {
+  align-items: center;
+  background-color: $orange;
+  box-shadow: 0 -8px 20px 0 #11111129;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  min-height: 20vh;
+  z-index: 1;
+
+  .footer-text {
+    color: $white;
+    font: $font;
+    font-size: 20px;
+    margin-top: 30px;
+  }
+
+  .footer-input {
+    border-radius: 5px;
+    margin-top: 20px;
+    min-height: 30px;
+    min-width: 300px;
```

Comment:
this would break the layout in smaller components.

use width: 100%; max-width: 300px instead.